robottelo/cli/fm_backup.py

```diff
@@ -0,0 +1,32 @@
+"""
+Usage:
+    foreman-maintain backup [OPTIONS] SUBCOMMAND [ARG] ...
+
+Parameters:
+    SUBCOMMAND                    subcommand
+    [ARG] ...                     subcommand arguments
+
+Subcommands:
+    online                        Keep services online during backup
+    offline                       Shut down services to preserve consistent backup
+    snapshot                      Use snapshots of the databases to create backup
+
+Options:
+    -h, --help                    print help
+"""
+from robottelo.cli.base import Base
+
+
+class Backup(Base):
+    """Manipulates Foreman-maintain's backup command"""
+
+    command_base = "backup"
+
+    @classmethod
+    def run_backup(cls, options=None, backup_dir='/tmp/', backup_type='online', timeout=None):
```

Comment:
I would move all none values parameters at the end and that also follows with FM backup command :

```suggestion
    def run_backup(cls, backup_type='online', backup_dir='/tmp/', options=None, timeout=None):
```

Non Blocking comment !